src/applications/covid-research-volunteer/sass/covid-research-volunteer.scss

```
@@ -6,3 +6,8 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../platform/forms/sass/m-form-confirmation";
+
+.input-width {
+  display: inline-block;
+  max-width: 20rem;
```

```suggestion
  max-width: units(15);
```
saw this recently on https://design.va.gov/design/spacing-units#using-the-spacing-tokens, not sure if it's helpful here or not